changelog.md

```diff
@@ -2,7 +2,7 @@
 
 | modName    | Taerobee (TBEE)                                                  |
 | ---------- | ---------------------------------------------------------------- |
-| license    | CC-BY-ND-3.0                                                     |
+| license    | CC-BY-NC-SA-4.0                                                  |
```

Comment:
The license is CC BY-ND 3.0 Unported, except for the localization config files - all submissions are subject to this license.